packages/dotcom-ui-header/src/header.scss

```diff
@@ -67,5 +67,34 @@
 }
 
 .ft-header__drawer-ask-ft-button {
-  margin-left: 16px;
+  display: block;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
+  text-align: center;
+
+}
+
+.edition-switcher {
+  display: flex;
+  align-items: center;
+  font-size: 16px;
```

Comment:
Do we need to set the font-size?

Reply:
yes because the default font-size of html is like 13px

Reply:
Default shroud be `16px`, but I don't know if there are any overrides happening.

Reply:
think you mentioned there is `18px` override